docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
-Review important information about the {kib} 8.0.0 releases.
+Review important information about the {kib} 8.x releases.
+
+* <<release-notes-8.8.0>>
+* <<release-notes-8.7.1>>
+* <<release-notes-8.7.0>>
+* <<release-notes-8.6.1>>
+* <<release-notes-8.6.0>>
+* <<release-notes-8.5.2>>
+* <<release-notes-8.5.1>>
+* <<release-notes-8.5.0>>
+* <<release-notes-8.4.3>>
+* <<release-notes-8.4.2>>
+* <<release-notes-8.4.1>>
+* <<release-notes-8.4.0>>
+* <<release-notes-8.3.3>>
+* <<release-notes-8.3.2>>
+* <<release-notes-8.3.1>>
+* <<release-notes-8.3.0>>
+* <<release-notes-8.2.3>>
+* <<release-notes-8.2.2>>
+* <<release-notes-8.2.1>>
+* <<release-notes-8.2.0>>
+* <<release-notes-8.1.3>>
+* <<release-notes-8.1.2>>
+* <<release-notes-8.1.1>>
+* <<release-notes-8.1.0>>
+* <<release-notes-8.0.0>>
+* <<release-notes-8.0.0-rc2>>
+* <<release-notes-8.0.0-rc1>>
+* <<release-notes-8.0.0-beta1>>
+* <<release-notes-8.0.0-alpha2>>
+* <<release-notes-8.0.0-alpha1>>
+
+--
+[[release-notes-8.8.0]]
+== {kib} 8.8.0
+
+coming::[8.8.0]
+
+Review the following information about the {kib} 8.8.0 release.
+
+[float]
+[[breaking-changes-8.8.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.8.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+[discrete]
+[[breaking-155470]]
+.Removes legacy project monitor API
+[%collapsible]
+====
+*Details* +
+The project monitor API for Synthetics in Elastic Observability has been removed. For more information, refer to {kibana-pull}155470[#155470].
+
+*Impact* +
+In 8.8.0 and later, an error appears when you use the project monitor API.
+====
+      
+[discrete]
+[[breaking-147985]]
+.Changes the privileges for alerts and cases
+[%collapsible]
+====
+*Details* +
+The privileges for attaching alerts to cases has changed. For more information, refer to {kibana-pull}147985[#147985].
+
+*Impact* +
+To attach alerts to cases, you must have `Read` access to Observability or Security, and `Write` access to the case. For detailed information, check link:https://www.elastic.co/guide/en/kibana/current/kibana-privileges.html[{kib} privileges] and link:https://www.elastic.co/guide/en/kibana/current/setup-cases.html[Configure access to cases]. 
```

Comment:
If you prefer that I update the source PR, please just let me know since I'm not sure where this was derived from:
```suggestion
To attach alerts to cases, you must have `Read` access to an {observability} or Security feature that has alerts and `All` access to the **Cases** feature. For detailed information, check <<kibana-privileges.html>> and <<setup-cases>>. 
```